UnityProject/Assets/Scripts/UI/Systems/AdminTools/DevTools/Search/DevSpawnerDocument.cs

```diff
@@ -15,12 +18,18 @@ public struct DevSpawnerDocument
 		/// <summary>
 		/// Name cleaned up for searchability (like lowercase).
 		/// </summary>
-		public readonly string SearchableName;
+		public readonly List<string> SearchableName;
 
 		private DevSpawnerDocument(GameObject prefab)
 		{
 			Prefab = prefab;
-			SearchableName = SpawnerSearch.Standardize(prefab.name);
+			var possibleNames = new List<string>();
+			possibleNames.Add(SpawnerSearch.Standardize(prefab.name));
+			if (prefab.TryGetComponent<PrefabTracker>(out var tracker))
+			{
+				if(string.IsNullOrWhiteSpace(tracker.AlternativePrefabName) == false) possibleNames.Add(tracker.AlternativePrefabName);
+			}
+			SearchableName = possibleNames;
```

Comment:
```suggestion
			SearchableName.Add(SpawnerSearch.Standardize(prefab.name));
			if (prefab.TryGetComponent<PrefabTracker>(out var tracker))
			{
				if(string.IsNullOrWhiteSpace(tracker.AlternativePrefabName) == false) 
				SearchableName .Add(tracker.AlternativePrefabName);
			}
```
Not sure why you wanted to make a new list?

Reply:
![image](https://user-images.githubusercontent.com/34368774/203542724-cccb3418-1e92-44e8-bff6-9a19f5b65569.png)

This is why

Reply:
Currently it's: `public readonly List<string> SearchableName;`

Needs to be initialised `public readonly List<string> SearchableName = new List<string>();`

Reply:
![image](https://user-images.githubusercontent.com/34368774/203545311-48d3adb8-0421-4c8b-98f6-4293d5fdb148.png)

you cant

Reply:
Ah its a struct. Guess your way works fine then or you can do:

```suggestion
			SearchableName = new List<string>();
			SearchableName.Add(SpawnerSearch.Standardize(prefab.name));
			if (prefab.TryGetComponent<PrefabTracker>(out var tracker))
			{
				if(string.IsNullOrWhiteSpace(tracker.AlternativePrefabName) == false) 
                                SearchableName.Add(tracker.AlternativePrefabName);
			}
```

I'll let you decide which code you'd prefer

Reply:
Done.